media/components/sdfv/index.html

```diff
@@ -8,10 +8,13 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>SDFG Viewer</title>
-    <link rel="stylesheet" type="text/css" href="{{ CSP_SRC }}/lib/bootstrap5/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="{{ CSP_SRC }}/lib/combobox/jquery-editable-select.min.css"></script>
-    <link rel="stylesheet" type="text/css" href="{{ CSP_SRC }}/webclient/external_lib/material/material-icons.css">
-    <link rel="stylesheet" type="text/css" href="{{ CSP_SRC }}/webclient/sdfv.css">
+
+    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/jquery-editable-select/dist/jquery-editable-select.min.css"></script>
+    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/material-icons/iconfont/material-icons.css">
+
+    <link rel="stylesheet" type="text/css" href="{{ NODE_MODULES }}/@spcl/sdfv/sdfv.css">
```

Comment:
Are you sure that the at symbol is ok here?

Reply:
In most web-contexts I wouldn't be ok with it, but electron doesn't complain.. :-)